tic.R

```diff
@@ -56,6 +56,7 @@ if (Sys.getenv("TUTORIAL") == "HTML") {
       add_step(step_setup_ssh())
 
     get_stage("deploy") %>%
+      add_code_step(devtools::document()) %>%
```

Comment:
This seems unrelated -- is it meant to be here?

Reply:
Yes, its indeed needed. Even though `pkdown::build_site()` writes NAMESPACE before the call, we get NS errors. So somehow the NS change is not picked up.

It only works if we explicitly create the NS using `devtools::document()`.

In this case this happened because I removed `getFilterValues()` entirely in this PR. Then pkgdown complained about the functions still being listed as "exported" in the NS and errored. Adding this step before solved the problem.